### DISCUSSION TITLE
How to use "prefer-workspace-packages"

### DISCUSSION THREAD

**Discussion by StarpTech** [pnpm/pnpm#4853] [Q&A] [answered]
Hi, is it configured in the "pnpm-workspace-yaml" or .npmrc? Thanks.

Related: https://github.com/pnpm/pnpm/issues/4656#issuecomment-1116337890
Github: https://github.com/search?p=1&q=prefer-workspace-packages&type=Code

**Comment by zkochan** [comment, marked as answer]
`.npmrc`
